### DISCUSSION TITLE
What comes next in this sequence?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#813] [Galaxy Trivia] [answered]
```
iron, bromine, uranium, argon, ???
```

**Comment by ParanoidUser** [comment]
@Eli-Softserve / @maxschneidercodes, any ideas? 😶‍🌫️

**Comment by SANKARAMDAS** [comment, marked as answer]
Yttrium; from the atomic element's symbols which are spelling February: Fe + Br + U + Ar + Y.
 iron (Fe), bromine (Br), uranium (U) , argon (Ar), yttrium(Y)

**Comment by peterbaccaro** [comment]
**Yttrium**; from the atomic element's symbols which are spelling February: Fe + Br + U + Ar + Y.
